src/fpm_filesystem.F90

```diff
@@ -1020,4 +1021,65 @@ subroutine get_home(home, error)
         end if
     end subroutine get_home
 
+    !> Execute command line and return output as a string.
```

Comment:
I think this should be built into fortran just as `execute_command_line`. I was searching whether we already have a routine implemented in `fpm` but couldn't find any. Have I missed sth? @awvwgk @everythingfunctional @milancurcic

Reply:
I haven't found any better method of capturing external command output in Fortran. I could potentially mention it to the standards committee.

Reply:
I think it would be great if we had that.